README.md

```diff
@@ -634,31 +634,27 @@ When running a binary (`pipx run BINARY`), pipx will
 These are all things you can do yourself, but pipx automates them for you. If you are curious as to what pipx is doing behind the scenes, you can always pass the `--verbose` flag to see every single command and argument being run.
 
 ## Contributing
-To develop `pipx` first clone the repository, then create and activate a virtual environment.
+To develop `pipx` first clone the repository, have tox with tox-venv installed somewhere, then run tox.
 ```
-python3 -m venv venv
-source venv/bin/activate
+tox --notest
 ```
-Next install pipx in "editable mode".
+
+Tox creates an environment in .tox/python. You may activate it thus.
 ```
-pip install -e .
+source .tox/python/bin/activate
 ```
 Now make your changes and run `pipx` as you normally would. Your changes will be used as soon as they are saved.
```

Comment:
I don't think this is true anymore since `pip install -e .` was removed. These docs should separate out making changes to pipx and having them reflected when running `pipx`, and running tox-based tests.

Reply:
The `usedevelop = True` in the tox.ini effectively causes `pip install -e` to the target environment(s)... so the changes should be reflected immediately to the version of pipx installed in that environment.